classes/admin.php

```diff
@@ -328,8 +328,19 @@ public function data($type, array $post = [])
             $type = preg_replace('|config/|', '', $type);
             $blueprints = $this->blueprints("config/{$type}");
             $config = $this->grav['config'];
-            $obj = new Data\Data($config->get($type, []), $blueprints);
+
+            if ($type === 'media') {
+                if (count($post) === 0) { //reading
+                    $obj = new Data\Data($config->get($type, []), $blueprints);
+                } else { //saving
+                    $obj = new Data\Data([], $blueprints);
+                }
```

Comment:
This could probably just be 

```php
if ($type === 'media' && count($post)) {
    $obj = new Data\Data([], $blueprints); // saving
} else {
    $obj = new Data\Data($config->get($type, []), $blueprints);
}
```